### ISSUE TITLE
One comment per params not align 1.0~

### ISSUE THREAD

**Issue by d3trax** [ongr-io/ongr-strict-standard#105]
In this situation when I have only one comment on parameters I got warning:
The comments for parameters $input_data (1) and $period (2) do not align

```
    /**
     * Calculate moving average.
     *
     * @param int[] $input_data
     * @param int   $period Smoothing window.
     * @param int   $xEnd
```

same for

```
    /**
     * Calculate moving average.
     *
     * @param int[] $input_data
     * @param int   $period Smoothing window.
     * @param int   $xEnd   Stop count.
```

**Comment by tautrimas** [comment]
I believe, this issue should be attributed upstream to https://github.com/squizlabs/PHP_CodeSniffer itself.

**Comment by mansimas** [comment]
```
    /**
     * Calculate moving average.
     *
     * @param int[] $input_data
     * @param int   $period Smoothing window.
     * @param int   $xEnd   Stop count.
```
It is fixed in 2.0.
Should be aligned to  $input_data (longest parameter name of all parameters in list). Parameter comments not required, but if exist, it must be aligned as after longest parameter in list.
```
    /**
     * Calculate moving average.
     *
     * @param int[] $input_data
     * @param int   $period      Smoothing window.
     * @param int   $xEnd        Stop count.
```
